perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/GroupsManagerBlImpl.java

```diff
@@ -3353,41 +3347,19 @@ private void synchronizeSubGroupsMembers(PerunSession sess, Group baseGroup, Ext
 			groupMemberExtsource.setValue(baseMemberExtsource.getValue());
 			getPerunBl().getAttributesManagerBl().setAttribute(sess, group, groupMemberExtsource);
 
-			synchronizeMembersAndSaveResult(sess, group);
-		}
+			Attribute isThisLightweightSynchro = getPerunBl().getAttributesManagerBl().getAttribute(sess, baseGroup, GroupsManager.GROUPLIGHTWEIGHTSYNCHRONIZATION_ATTRNAME);
+			isThisLightweightSynchro.setValue(lightWeightSynchronization.getValue());
+			getPerunBl().getAttributesManagerBl().setAttribute(sess, group, isThisLightweightSynchro);
 
-	}
+			Attribute synchroInterval = getPerunBl().getAttributesManagerBl().getAttribute(sess, baseGroup, GroupsManager.GROUPSYNCHROINTERVAL_ATTRNAME);
+			synchroInterval.setValue(synchronizationInterval.getValue());
+			getPerunBl().getAttributesManagerBl().setAttribute(sess, group, synchroInterval);
 
-	/**
-	 * Synchronize members under group and save information about it
-	 *
-	 * Method used by group structure synchronization
-	 *
-	 * @param sess perun session
-	 * @param group under which will be members synchronized
-	 */
-	private void synchronizeMembersAndSaveResult(PerunSession sess, Group group) {
-		String exceptionMessage = null;
-		String skippedMembersMessage = null;
-		boolean failedDueToException = false;
-		try {
-			List<String> skippedMembers = perunBl.getGroupsManagerBl().synchronizeGroup(sess, group);
-			skippedMembersMessage = prepareSkippedObjectsMessage(skippedMembers, "members");
-			exceptionMessage = skippedMembersMessage;
-
-		} catch (Exception e) {
-			failedDueToException = true;
-			exceptionMessage = "Cannot synchronize group ";
-			log.error(exceptionMessage + group, e);
-			exceptionMessage += "due to exception: " + e.getClass().getName() + " => " + e.getMessage();
-		} finally {
-			try {
-				perunBl.getGroupsManagerBl().saveInformationAboutGroupSynchronizationInNestedTransaction(sess, group, failedDueToException, exceptionMessage);
-			} catch (Exception ex) {
-				log.error("When synchronization group " + group + ", exception was thrown.", ex);
-				log.info("Info about exception from synchronization: " + skippedMembersMessage);
-			}
+			Attribute synchroEnabled = getPerunBl().getAttributesManagerBl().getAttribute(sess, baseGroup, GroupsManager.GROUPSYNCHROENABLED_ATTRNAME);
+			synchroEnabled.setValue("true");
```

Comment:
You can get the attribute before the 'for' loop as well.

Reply:
fixed